tox.ini

```diff
@@ -104,3 +104,11 @@ commands =
     coverage html -d {toxworkdir}/htmlcov -i
     python -m diff_cover.diff_cover_tool --compare-branch {env:DIFF_AGAINST:origin/main} {toxworkdir}/coverage.xml
 depends = {py312, py311, py310, py39, py38, py37, pypy37, pypy38, pypy39}{,-min}, path
+
+[testenv:bump]
+description = bump versions, pass major/minor/patch
+skip_install = true
+setenv =
```

Comment:
set_env

Reply:
:facepalm: Ahh, I should have noticed that.

Reply:
Ahh, I was copying the examples above, which also use the older format. Maybe we should move to the newer more readable forms all at once?

Reply:
Use https://github.com/tox-dev/tox-ini-fmt that auto upgrades

Reply:
That didn't add the underscores, all fields are the same after running it. It just moved things around and added some indentation. And broke this:

```diff
-depends = {py312, py311, py310, py39, py38, py37, pypy37, pypy38, pypy39}{,-min}, path
+depends =
+    -min}
+    path
+    py310
+    py311
+    py37
+    py38
+    py39
+    pypy37
+    pypy38
+    pypy39}{
+    {py312
```

Reply:
<img width="704" alt="image" src="https://github.com/pypa/build/assets/690238/de0c309f-7c94-42c3-86b2-0db379cc627f">

Are you using the latest?

Reply:
Fixed https://github.com/tox-dev/tox-ini-fmt/commit/69ba5f57bfbd234f3de4d15275e8122526b4d80f 👍